eiffel-vocabulary/EiffelConfigurationChangedEvent.md

```diff
@@ -0,0 +1,54 @@
+# EiffelConfigurationChangedEvent
+The EiffelConfigurationChangedEvent signifies that the configuration of an item, typically an actor within the continuous integration and delivery pipeline, has changed.
+
+Similarly to [EiffelDocumentationCreatedEvent](EiffelDocumentationCreatedEvent.md), this event is intended for situations where configuration is not stored and treated as any other source (in which case [EiffelSourceChangeCreatedEvent](EiffelSourceChangeCreatedEvent.md) and [EiffelSourceChangeSubmittedEvent](EiffelSourceChangeSubmittedEvent.md) SHOULD be used instead of EiffelConfigurationChangedEvent).
+
+## Data Members
+### data.item
+__Type:__ Object  
+__Required:__ Yes  
+__Description:__ The item the configuration of which has changed.
+
+#### data.item.type
+__Type:__ String  
+__Required:__ No  
+__Description:__ The type of reconfigured item.
+
+#### data.item.name
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The name of the reconfigured item.
+
+#### data.item.uri
+__Type:__ String  
+__Required:__ No  
+__Description:__ The URI of the reconfigured item.
+
+### data.configuration
+__Type:__ Object  
+__Required:__ Yes  
+__Description:__ The changed configuration.
+
+#### data.configuration.change
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ CREATED, MODIFIED, DELETED
+__Description:__ The type of change made to the configuration.
+
+#### data.configuration.uri
```

Comment:
I think that would be an improvement.

Do we need to consider the "item" we are applying the configuration on. Consider this:
creating/removing a job in Jenkins would apply the configuration on the Jenkins server (data.item.uri=http://my.jenkins.server/jenkins). Later when the job definition is modified it would be applied on the job (data.item.uri=http://my.jenkins.server/jenkins/myJob).

Do we need to start thinking about the topology of the items?